packages/biometric-ed25519/src/index.ts

```diff
@@ -129,3 +139,13 @@ export const getKeys = async (username: string): Promise<[KeyPair, KeyPair]> =>
 export const isPassKeyAvailable = async (): Promise<boolean> => {
     return window.PublicKeyCredential?.isUserVerifyingPlatformAuthenticatorAvailable?.() || false;
 };
+
+// To check if current device is supported
```

Comment:
Can this comment please be updated to explain _what_ the device does or doesn't support? This is not immediately clear from the PR; as it stands this comment is not super descriptive 🙂